de_web_plugin.cpp

```diff
@@ -761,9 +761,9 @@ DeRestPluginPrivate::DeRestPluginPrivate(QObject *parent) :
     }
 
     const QStringList buttonMapLocations = {
-        deCONZ::getStorageLocation(deCONZ::ApplicationsDataLocation) + QLatin1String("/devices/button_maps.json")
 #ifdef Q_OS_LINUX
-        , "/usr/share/deCONZ/devices/button_maps.json"
+        QLatin1String("/usr/share/deCONZ/devices/button_maps.json"), deCONZ::getStorageLocation(deCONZ::ApplicationsDataLocation) +
+        QLatin1String("/devices/button_maps.json")
```

Comment:
I think this change is problematic.

`deCONZ::getStorageLocation(deCONZ::ApplicationsDataLocation) +  QLatin1String("/devices/button_maps.json")`

- This line should always come first so it has a higher priority as the packaged button_maps.json;
- It's not Linux specific, with this change it wouldn't work on Windows anymore.

Reply:
Change's reverted